components/nexus-repository/src/main/java/org/sonatype/nexus/repository/search/DefaultComponentMetadataProducer.java

```diff
@@ -65,7 +70,7 @@ public String getMetadata(final Component component) {
     }
 
     try {
-      return new ObjectMapper().writerWithDefaultPrettyPrinter().writeValueAsString(metadata);
+      return objectMapper.writerWithDefaultPrettyPrinter().writeValueAsString(metadata);
```

Comment:
minor but you can likewise store the prettyPrinter object which is otherwise instantiated per call (it is threadsafe as well)